src/components/common/NavigationBar.tsx

```
@@ -0,0 +1,59 @@
+    <nav
+      className={cn(
+        "rounded-t-20 flex w-full justify-between bg-gray-900 px-5 pt-3.75 pb-7.75 [border-top:0.4px_solid_var(--color-gray-800)]",
+        className,
+      )}>
+      {NAV_ITEMS.map(({ href, label, icon: Icon, iconClassName = "size-6" }) => {
+        const isActive = href === "/" ? pathname === "/" : pathname.startsWith(href);
+        return (
+          <Link
+            key={href}
+            href={href}
+            className={cn("flex flex-col items-center transition-all duration-300 hover:text-white", isActive ? "text-white" : "text-gray-700")}>
```

ESLint

[error] Replace `"flex·flex-col·items-center·transition-all·duration-300·hover:text-white",·isActive·?·"text-white"·:·"text-gray-700"` with `⏎··············"flex·flex-col·items-center·transition-all·duration-300·hover:text-white",⏎··············isActive·?·"text-white"·:·"text-gray-700",⏎············`

(prettier/prettier)